fastlane/Fastfile

```diff
@@ -183,12 +184,23 @@ platform :ios do
 
   desc "Run StoreKitTests"
   lane :storekit_tests do |options|
+    ensure_testing_environment
     replace_api_key_integration_tests
     scan(scheme: "StoreKitTests", derived_data_path: "scan_derived_data")
   end
 
 end
 
+# Should be called before any tests take place to ensure tests can actually run/fastlane can finish successfully.
+def ensure_testing_environment
+  if ENV["REVENUECAT_API_KEY"].to_s.strip.empty?
+    raise "Missing REVENUECAT_API_KEY environment variable 😵."
+  end
+  if ENV["REVENUECAT_PROXY_URL"].to_s.strip.empty?
+    raise "Missing REVENUECAT_PROXY_URL environment variable 😵."
+  end
```

Comment:
this is good. However the proxy url is actually not in use for some things right now, since we don't have a proper staging environment for some features.
So right now these would fail